packages/x-date-pickers/src/DateTimePicker/DateTimePickerToolbar.tsx

```diff
@@ -30,6 +31,15 @@ export interface DateTimePickerToolbarProps<TDate>
    */
   classes?: Partial<DateTimePickerToolbarClasses>;
   toolbarVariant?: WrapperVariant;
+  /**
+   * If `true`, will toggle the `selected` state for the currently active view.
+   * Used by `DateTimeRangePickerToolbar` to mark only the currently selected `rangePosition` as active.
```

Comment:
I'm honestly fine with this JSDoc

Reply:
@flaviendelangle WDYT about this solution: https://github.com/mui/mui-x/pull/9528/commits/0076c4930d94cc04371d138d7e5ca93a09f88d27 to avoid the need for `markSelected` altogether? 🤔

Reply:
It's slightly cleaner :partying_face: